### ISSUE TITLE
A11y_AzureToolsStorageExplorer_AddCORSRule_ScreenReader: VoiceOver not announces Allowed methods edit box name.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1969]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac;  

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Configure CORS Settings" and hit enter.
5. Navigate to "Add" control and hit enter, "Add CORS Rule" screen gets opened.
6. Start VoiceOver and navigate to Allowed methods edit box by using Tab key.
7. Verify VoiceOver not announces Allowed methods edit box name.

**​Actual:​**
While navigating on the screen using Tab key when Voice Over focus is on "Allowed Methods" edit box, VoiceOver not announces name of the edit box.

**Expected:**
VoiceOver should announce name of the edit box as "Allowed Methods".

**UserImpact:**
Screen reader users may not be able to know the name of the edit box if screen reader not announces edit box name for them.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[1969_A11y_AzureToolsStorageExplorer_AddCORSRule_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669413/1969_A11y_AzureToolsStorageExplorer_AddCORSRule_ScreenReader.pptx)

**Comment by craxal** [comment]
It seems the issue here is that inputs with text auto-select the text when they receive focus, so VoiceOver doesn't read the control name, just the selected text. This looks like a problem with just VoiceOver, because NVDA doesn't do this.

**Comment by craxal** [comment]
@Suraj1200499 Please comment with further feedback. Dev team cannot progress on our end since this is an external issue.

**Comment by Suraj1200499** [comment]
@Craxal 
Apologies for delay.
I have to put this issue in note of our SME's, Once I get reply from them, I will update the comments ASAP.

**Comment by MRayermannMSFT** [comment]
@craxal , please test this bug and see if it repro's in Chrome. If it does, then you can just open an issue there and we can use that in the VPAT.

**Comment by craxal** [comment]
Yes, this reproduces on Chromium (tested on latest Edge-Developer).

Chromium bug reported: https://bugs.chromium.org/p/chromium/issues/detail?id=1024011

**Comment by Suraj1200499** [comment]
@Craxal Since the primary bug#[1024011](https://bugs.chromium.org/p/chromium/issues/detail?id=1024011) is in open State. We will take action on the bug#[1969](https://github.com/microsoft/AzureStorageExplorer/issues/1969) Once the Primary bug is Closed.

**Comment by Alexandria-R** [comment]
Approved as low impact

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
